internal/driver/glfw/window_desktop.go

```diff
@@ -297,6 +301,13 @@ func (w *window) getMonitorForWindow() *glfw.Monitor {
 }
 
 func (w *window) detectScale() float32 {
+	// check if DPI detection is disabled
+	env := os.Getenv(disableDPIDetectionEnvKey)
+	switch env {
+	case "True", "TRUE", "true", "T", "t", "1":
```

Comment:
You might want to use `strings.EqualFold` instead of hardcoding values. That should allow the word "true" no matter how it is capitalised.